patches/hydrogen-cmake.patch

```diff
@@ -0,0 +1,11 @@
+--- CMakeLists.txt	2018-03-18 07:50:43.000000000 -0400
```

Comment:
Has it been upstreamed?

Reply:
not yet.

Reply:
A variant has been upstreamed hydrogen-music/hydrogen#689